docs/api/aave_v2_stablecoin.md

```diff
@@ -0,0 +1,199 @@
+# Protocol Documentation
+<a name="top"></a>
+
+## Table of Contents
+
+- [aave_v2_stablecoin.proto](#aave_v2_stablecoin-proto)
+    - [AaveV2Stablecoin](#steward-v1-AaveV2Stablecoin)
+    - [AaveV2Stablecoin.AccrueFees](#steward-v1-AaveV2Stablecoin-AccrueFees)
+    - [AaveV2Stablecoin.ClaimAndUnstake](#steward-v1-AaveV2Stablecoin-ClaimAndUnstake)
+    - [AaveV2Stablecoin.EnterPosition](#steward-v1-AaveV2Stablecoin-EnterPosition)
+    - [AaveV2Stablecoin.Rebalance](#steward-v1-AaveV2Stablecoin-Rebalance)
+    - [AaveV2Stablecoin.Rebalance.SwapParams](#steward-v1-AaveV2Stablecoin-Rebalance-SwapParams)
+    - [AaveV2Stablecoin.Reinvest](#steward-v1-AaveV2Stablecoin-Reinvest)
+    - [AaveV2Stablecoin.Sweep](#steward-v1-AaveV2Stablecoin-Sweep)
+    - [AaveV2Stablecoin.TransferFees](#steward-v1-AaveV2Stablecoin-TransferFees)
+  
+- [Scalar Value Types](#scalar-value-types)
+
+
+
+<a name="aave_v2_stablecoin-proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## aave_v2_stablecoin.proto
+
+
+
+<a name="steward-v1-AaveV2Stablecoin"></a>
+
+### AaveV2Stablecoin
+Represents a function call to the Aave V2 Stablecoin cellar
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| accrue_fees | [AaveV2Stablecoin.AccrueFees](#steward-v1-AaveV2Stablecoin-AccrueFees) |  |  |
+| claim_and_unstake | [AaveV2Stablecoin.ClaimAndUnstake](#steward-v1-AaveV2Stablecoin-ClaimAndUnstake) |  |  |
+| enter_position | [AaveV2Stablecoin.EnterPosition](#steward-v1-AaveV2Stablecoin-EnterPosition) |  |  |
+| rebalance | [AaveV2Stablecoin.Rebalance](#steward-v1-AaveV2Stablecoin-Rebalance) |  |  |
+| reinvest | [AaveV2Stablecoin.Reinvest](#steward-v1-AaveV2Stablecoin-Reinvest) |  |  |
+| sweep | [AaveV2Stablecoin.Sweep](#steward-v1-AaveV2Stablecoin-Sweep) |  |  |
+| transfer_fees | [AaveV2Stablecoin.TransferFees](#steward-v1-AaveV2Stablecoin-TransferFees) |  |  |
+
+
+
+
+
+
+<a name="steward-v1-AaveV2Stablecoin-AccrueFees"></a>
+
+### AaveV2Stablecoin.AccrueFees
+Take platform fees and performance fees off of cellar&#39;s active assets.
+
+Represents function `accruePlatformFees()``
```

Comment:
Remove extra tick from `accruePlatformFees()`

Reply:
Overall, I think the `label` field should be expunged from the docs since it's empty

Reply:
Label is used in a couple of place, not much though. These are auto-generated so I'm not sure it's a good idea to edit them manually since we'd have to do it every time we make changes to the proto or proto comments.